ts/src/binance.ts

```diff
@@ -233,6 +233,7 @@ export default class binance extends Exchange {
                         'asset/convert-transfer/queryByPage': 0.033335,
                         'asset/wallet/balance': 6, // Weight(IP): 60 => cost = 0.1 * 60 = 6
                         'asset/custody/transfer-history': 6, // Weight(IP): 60 => cost = 0.1 * 60 = 6
+                        'spot/delist-schedule': 10, // Weight(IP): 100 => cost = 0.1 * 100 = 10
```

Comment:
@sc0Vu is your build and linter working well locally? 
I added this yesterday 
![image](https://github.com/ccxt/ccxt/assets/43336371/b7aec0b6-5eee-443a-8a7b-ff67137b9f68)